lib/mix/tasks/phx.gen.channel.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Mix.Tasks.Phoenix.Gen.Channel do
```

Comment:
@chrismccord I believe this should be `Mix.Tasks.Phx.Gen.Channel` to match the filename and the `@moduledoc` below should be updated as well